docs/PUBLISHING.md

```diff
@@ -93,7 +93,7 @@ access resin.io's production downloads S3 bucket.
 Run the following command to publish a specific file:
 
 ```sh
-./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -t <production|snapshot>
+./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -p <product name>
```

Comment:
This document describes `make publish-aws-s3 RELEASE_TYPE=<production|snapshot>` right below, so I guess that covers the snapshot part already.